package.json

```diff
@@ -10,15 +10,27 @@
     "@types/node": "^12.12.57",
     "@types/react": "^16.9.49",
     "@types/react-dom": "^16.9.8",
+    "@types/react-router": "^5.1.8",
+    "@types/react-router-dom": "^5.1.5",
+    "@typescript-eslint/eslint-plugin": "4.1.0",
+    "@typescript-eslint/parser": "4.1.0",
+    "d3-scale": "^3.2.2",
+    "es6-tween": "^5.5.11",
     "eslint": "6.6.0",
     "eslint-plugin-react": "7.20.6",
-    "@typescript-eslint/parser": "4.1.0",
-    "@typescript-eslint/eslint-plugin": "4.1.0",
+    "node-sass": "4.14.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-globe": "^5.0.2",
+    "react-router": "5.2.0",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
+    "resize-observer-polyfill": "^1.5.1",
+    "three": "^0.120.1",
+    "three-glow-mesh": "^0.1.2",
+    "three.interaction": "^0.2.3",
+    "tippy.js": "^6.2.6",
```

Comment:
we don't seem to use many of these... do we need them all?

Reply:
Full list of react-globe dependencies are [here](https://www.npmjs.com/package/react-globe), seems to break without them!